src/encoded/static/components/item-pages/components/ItemFooterRow.js

```diff
@@ -0,0 +1,315 @@
+
+    const types = context && context['@type'] && Array.isArray(context['@type']) ? context['@type'] : [];
+    const renderAsTable = (types.indexOf('ExperimentSet') > -1 || types.indexOf('Experiment') > -1);
+
+    if (renderAsTable) {
+        const altAccessionTitle = (<h3 className="tab-section-title">Alternate Accessions</h3>);
+        return (
+            <div className="row">
+                <ExternalReferencesStackedTable {...{ context, width }} />
+                {/* <AliasesSection context={context} aliases={aliases} actions={actions} /> */}
+                <AlternateAccessionSection context={context} alternateAccessions={alternate_accessions} title={altAccessionTitle} />
+            </div>
+        );
+    } else {
+        if (external_references.length === 0 && alternate_accessions.length === 0) {
+            return null;
+        }
+        return (
+            <React.Fragment>
+                <hr className="mb-08 mt-1" />
+                <div className="row">
+                    <ExternalReferencesSection context={context} externalReferences={external_references} />
+                    {/* <AliasesSection context={context} aliases={aliases} actions={actions} /> */}
+                    <AlternateAccessionSection context={context} alternateAccessions={alternate_accessions} />
+                </div>
+            </React.Fragment>
+        );
+    }
+});
+
+
+function ExternalReferencesSection({ externalReferences }){
+    if (externalReferences.length === 0){
+        return null;
+    }
+    return (
+        <div className="col col-12 col-md-6">
+            <h4 className="text-300">External References</h4>
+            <div>
+                <ul>
+                    { _.map(externalReferences, function(extRef, i){
+                        return (
+                            <li key={i}>
+                                { typeof extRef.ref === 'string' ?
+                                    <ExternalReferenceLink uri={extRef.uri || null}>{ extRef.ref }</ExternalReferenceLink> : extRef
+                                }
+                            </li>
+                        );
+                    }) }
+                </ul>
+            </div>
+        </div>
+    );
+}
+
+function AlternateAccessionSection({ alternateAccessions, title }){
+    if (alternateAccessions.length === 0){
+        return null;
+    }
+    return (
+        <div className="col col-12 col-md-6">
+            {title || (<h4 className="text-300">Alternate Accessions</h4>)}
+            <div>
+                <ul>
+                    { _.map(alternateAccessions, function(altAccession, i){
+                        return (
+                            <li key={i}>{ altAccession }</li>
+                        );
+                    }) }
+                </ul>
+            </div>
+        </div>
+    );
+}
+
+function AliasesSection({ aliases, actions }){
```

Comment:
Codacy found an issue: ['AliasesSection' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5995039)